frontend/src/lib/utils/candidateStore.ts

```diff
@@ -31,10 +41,12 @@ export interface CandidateContext {
 const userStore = writable<User | null>(null);
 const tokenStore = writable<string | null | undefined>(undefined);
 const emailOfNewUserStore = writable<string | undefined>(undefined);
-const answersStore = writable<Record<string, Answer> | undefined>(undefined);
-const questionsStore = writable<Record<string, Question> | undefined>(undefined);
+const opinionAnswerStore = writable<Record<string, Answer> | undefined>(undefined);
+const infoAnswerStore = writable<Record<string, CandidateAnswer> | undefined>(undefined);
+const opinionQuestionsStore = writable<Record<string, Question> | undefined>(undefined);
+const infoQuestionsStore = writable<QuestionProps[] | undefined>(undefined);
 const basicInfoFilledStore = writable<boolean | undefined>(undefined);
-const nofUnansweredBasicInfoQuestionsStore = writable<number | undefined>(undefined);
+const nofUnansweredInfoQuestionsStore = writable<number | undefined>(undefined);
```

Comment:
Consider the suggestions earlier for `CandidateContextProvider.svelte` with this and related stores.